request.schema.json

```diff
@@ -39,6 +39,10 @@
             },
             "required": [ "latitude", "longitude" ]
         },
+        "placeID": {
```

Comment:
What actually is this ID? Where does it come from? Why is it required?

Reply:
We haven't gotten to the browser side for this yet, but it is the substitute for a lat/long. eg the map may be a query of the Eiffel Tower instead of a lat/long and therefore we would make ```placeID='Eiffel+Tower,Paris+France```

Reply:
So this is a google specific ID? Or is it just a query?

Reply:
It is called a placeID by Google and can be structured in plain text or by a unique code as I understand

Reply:
The embedded map request allows for an english like query or an ID (https://developers.google.com/maps/documentation/places/web-service/place-id)